docs/dbg.md

```diff
@@ -0,0 +1,162 @@
+# DBG helper
+
+
+### store
+
+`dbg.store` is a convenient way for getting the redux store.
+
+```js
+const store = dbg.store.getState()
+
+// select a source
+const source = dbg.store.getState().sources.sources.first();
+```
+
+### actions
+
+`dbg.actions` is a convenient way for getting the redux actions.
+The actions are bound so you can call them directly!
+
+```js
+const source = dbg.store.getState().sources.sources.first();
+dbg.actions.selectSource(source.get("id")))
+```
+
+### selectors
+
+`dbg.selectors` is a convenient way for getting the redux selectors.
+The selectors are bound so you can call them without `store.getState()`!
+
+```js
+const source = dbg.selectors.getSelectedSource();
+```
+
+### client
+
+`dbg.client` is a convenient way for getting the firefox commands.
+The commands are the interface for talking to the debugger server.
+
+```js
+const source = dbg.selectors.getSelectedSource();
+dbg.client
+  .setBreakpoint({line: 24, sourceId: source.get("id")})
+  .then(console.log)
+```
+
+### connection
+
+`dbg.connection` has several useful properties for seeing the current connection to the tab.
```

Comment:
maybe this could be clearer about what " connection" is (or what it is representing). At the moment I know from this that it has properties, but i don't know if it is something useful itself -- is like a grab-bag of these properties but not an actual connection instance, or is it a connection instance that reflects some data elsewhere is the question i find myself asking here